subsys/bluetooth/host/hci_core.c

```diff
@@ -4828,6 +4828,45 @@ static void le_past_received(struct net_buf *buf)
 		}
 	}
 }
+
+#if defined(CONFIG_BT_ISO_BROADCAST)
+static void hci_le_biginfo_adv_report(struct net_buf *buf)
+{
+	struct bt_hci_evt_le_biginfo_advertising_report *evt =
+		(struct bt_hci_evt_le_biginfo_advertising_report *)buf->data;
```

Comment:
This looks duplicated since you will be doing net_buf_pull_mem and overwriting the original pointer, Id leave just the later since it also adjust the buffer position.

Reply:
Nice catch :)